src/backend/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM continuumio/miniconda3
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+
+RUN conda create -n inatator python=3.10
+
+COPY . .
+
+RUN pip install -r requirements.txt && pip install -r requirements-dev.txt
```

Make sure that the dependencies get installed to the conda environment you created. Since we're only using the Docker containers for deployment, you don't need to install the dev dependencies here. 

```suggestion
RUN conda run -n inatator \
    pip install -r requirements.txt
```

---

Thanks for the catch, I added your suggestion and was able to successfully build